### PR TITLE
Add systemd service file, postinstall script for log dirs and fix typo in doc

### PR DESCRIPTION
**Pull request by nsteinmetz**
* Add systemd service file
* Add postinstall script to create log dirs
* Fix typo in doc

### CODE REVIEW

File: src/docs/content/docs/backends/cassandra.md [hunk 1]
```diff
@@ -23,8 +23,8 @@ cassandra:
 If you're using authentication or SSL:
 
 ```yaml
+storageType: cassandra
```
Comment (ossarga) [inline src/docs/content/docs/backends/cassandra.md:26]: Nice catch!

**Review by ossarga — COMMENTED** [review]
Changes LGTM.

**Review by ossarga — APPROVED** [review]
Changes LGTM

**Comment by michaelsembwever** [comment]
Manually merged.

**Pull request closed without merging**